packages/mobile/src/invite/saga.ts

```diff
@@ -48,59 +45,36 @@ export async function getInviteFee(
 export function* sendInvite(
   e164Number: string,
   amount: BigNumber,
-  currency: Currency,
+  tokenAddress: string,
   feeInfo?: FeeInfo
 ) {
   try {
     ValoraAnalytics.track(InviteEvents.invite_start, {
-      escrowIncluded: true,
       amount: amount.toString(),
+      tokenAddress,
     })
 
-    const web3: Web3 = yield call(getWeb3)
```

Comment:
I'm wondering, why this code is removed or why was it there in the first place... Just to understand it can you give me more context? :)

Reply:
Yes, so this code hasn't been used for a long time.

Originally, before we had Komenci, you needed to pay $0.15 to verify your phone number. So, if you wanted to invite someone you were asked to pay $0.15. The funds were put in a temporary account and an "invite code" was created. This invite code had access to the private key of the temporary address.

In the onboarding flow there was a screen where you could put in the code to get the $0.15 transferred from the temporary wallet to your new one you just created.

This stopped being used late last year when Komenci was implemented to pay for the verification fees, so that's the deleted code here :)